Resources/public/js/views/fields/ez-relation-editview.js

```diff
@@ -133,12 +133,23 @@ YUI.add('ez-relation-editview', function (Y) {
          * @param {EventFacade} e
          */
         _runUniversalDiscovery: function (e) {
+            var that = this;
```

Comment:
I agree with the consistent point, but since we wrote this code we tried to avoid using `this = that`.
I do think this would be better to write something like :
```js
isSelectable: Y.bind(function (contentStruct) {
    var selectionContentTypes = this.get('fieldDefinition').fieldSettings.selectionContentTypes,
          contentTypeIdentifier = contentStruct.contentType.get('identifier');
 
    return (
        (selectionContentTypes.length === 0)
        || (selectionContentTypes.indexOf(contentTypeIdentifier) > -1)
    );
}, this)
```

Furthermore I forgot to mention it before, but unit tests should also be updated.